fetch/api/resources/infinite-slow-response.py

```diff
@@ -12,24 +12,25 @@ def stash_write(request, key, value):
 
 
 def main(request, response):
-    stateKey = request.GET.first("stateKey", "")
-    abortKey = request.GET.first("abortKey", "")
+    stateKey = request.GET.first(b"stateKey", b"")
+    abortKey = request.GET.first(b"abortKey", b"")
 
     if stateKey:
         stash_write(request, stateKey, 'open')
 
-    response.headers.set("Content-type", "text/plain")
+    response.headers.set(b"Content-type", b"text/plain")
     response.write_status_headers()
 
     # Writing an initial 2k so browsers realise it's there. *shrug*
-    response.writer.write("." * 2048)
+    response.writer.write(u"." * 2048)
```

Comment:
```suggestion
    response.writer.write(b"." * 2048)
```
semantically more correct here